src/dotnet/Fable.Client.Browser/Fable.Client.Browser.fsproj

```diff
@@ -30,8 +28,7 @@
     <Compile Include="../Fable.Compiler/FSharp2Fable.Util.fs"/>
     <Compile Include="../Fable.Compiler/FSharp2Fable.fs"/>
     <Compile Include="../Fable.Compiler/Fable2Babel.fs"/>
-
-    <Compile Include="../Fable.Tools/State.fs"/>
+    <Compile Include="../Fable.Compiler/State.fs"/>
```

Comment:
@alfonsogarciacaro Yes, it does, I actually mean compile with dotnet. It helps a lot to be able to compile the same source with dotnet, for sanity check.